widget/svelte.config.js

```diff
@@ -0,0 +1,5 @@
+const sveltePreprocess = require('svelte-preprocess')
```

Comment:
It looks like this file may not be necessary at all: https://github.com/HellButcher/parcel-transformer-svelte3-plus#configuration

If we do need it, I'd rather use `.svelterc` as a declarative JSON format, instead of `svelte.config.js`, if possible.